dataland-frontend/src/components/forms/parts/UploadFormHeader.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="form-field-label">
     <h5>{{ name }}</h5>
+    <div class="asterisk" v-if="isRequired">*</div>
```

Comment:
from a semantic point of view, this should be rather a span - shouldn't it?

![image](https://user-images.githubusercontent.com/47553592/233922644-d684b534-5a5b-4227-b3f4-e574e85f50f8.png)

Reply:
Agree that for a single * a span is more appropriate. Has been changed accordingly.

Reply:
It has been confirmed that display has not been unexpectedly changed by switching from `span` to `div`